### DISCUSSION TITLE
SMS are not displayed on the map

### DISCUSSION THREAD

**Discussion by niang77** [ushahidi/platform#4256] [General]
Hello
I have configured my SMS frontline and the url and APIs are working fine.
But if I send a text message it does not appear on the map.
thank you for helping me

**Comment by CeciliaHinga** [comment]
Hello Niang77, have you set your deployment as private?

**Comment by niang77** [comment]
hello Ceciliahinga
how do we check if it's private or public?

**Comment by niang77** [comment]
ha okay i see 
the deployment is public

**Comment by CeciliaHinga** [comment]
Can we move this conversation to support@ushahidi.com? It will be easier for us to dig deeper into what the issue might be. Thanks

**Comment by tuxpiper** [comment]
Hi,

Just a comment that in most cases we've seen, SMS messages don't arrive on the platform with location data attached to them. As such, platform posts that originated from SMS only appear on the map if something or someone has pinpointed a location for them.

**Comment by niang77** [comment]
how do I put the location dots in the messages? 

    Le jeudi 20 mai 2021 à 13:38:53 UTC+2, David Losada ***@***.***> a écrit :  
 
 


Hi,

Just a comment that in most cases we've seen, SMS messages don't arrive on the platform with location data attached to them. As such, platform posts that originated from SMS only appear on the map if something or someone has pinpointed a location for them.

**Reply by tuxpiper** [reply]
If your SMS source is not configured to put the messages in a survey automatically, you would first assign each particular message to a survey. It's a process known as structuring, and here's the relevant manual section: https://docs.ushahidi.com/ushahidi-platform-user-manual/6.-managing-data-in-your-deployment/6.4-editing-posts#6-4-1-structuring-posts 

You would, of course, need to assign the message to a survey that has a location field in it.

I hope this helps!

**Comment by niang77** [comment]
It's too complicated for me.I thought that with fronline cloud the messages that we send will be displayed directly on the map. 

    Le jeudi 20 mai 2021 à 13:50:44 UTC+2, David Losada ***@***.***> a écrit :  
 
 


If your SMS source is not configured to put the messages in a survey automatically, you would first assign each particular message to a survey. It's a process known as structuring, and here's the relevant manual section: https://docs.ushahidi.com/ushahidi-platform-user-manual/6.-managing-data-in-your-deployment/6.4-editing-posts#6-4-1-structuring-posts

You would, of course, need to assign the message to a survey that has a location field in it.

I hope this helps!

**Reply by tuxpiper** [reply]
I want to double-check because what you say is very interesting. Maybe there's something we missed.

Are you aware if Frontline does have specific features to assign source locations to SMS messages?

**Comment by niang77** [comment]
What I want is how to integrate location on sent messagesSo that it appears on my websiteThank you 

    Le jeudi 20 mai 2021 à 14:08:07 UTC+2, David Losada ***@***.***> a écrit :  
 
 


I want to double-check because what you say is very interesting. Maybe there's something we missed.

Are you aware if Frontline does have specific features to assign source locations to SMS messages?
